README.md

```diff
@@ -6,6 +6,17 @@ troubleshooting the host operating system, without having to install software
 on the host. It is built on top of [Podman](https://podman.io/) and other
 standard container technologies from [OCI](https://opencontainers.org/).
 
+> [!NOTE]
+> **Toolbx's original name was "toolbox".** Work is still in progress to
+> update the old name to the new in various places, thus the name of the
+> package and the binary may still be `toolbox` on various systems.
```

Comment:
The binary _is still_ called `toolbox`, not _may still be_.  It is one of the most visible obstacles.  :)